### DISCUSSION TITLE
How to know my node miss some rewards?

### DISCUSSION THREAD

**Discussion by irocn** [Chia-Network/chia-blockchain#15417] [Support]
Hello,

My node have 1.5P power but don't get any rewards after 10 days. I want to know how to find any issues if miss rewards? can find any clue from debug.log? or any keywords is help in log file? Thanks!

**Comment by iTZUNAMI** [comment]
Config File

In config.yaml you can set the level of detail for your logs.

Look for this section in config.yaml. It’s useful to change the logger setting log_level from WARNING to INFO to get the detail needed to troubleshoot.


info: https://github.com/Chia-Network/chia-blockchain/wiki/How-to-Check-If-Everything-is-Working-(or-Not)
